solution.md

```diff
@@ -0,0 +1,17 @@
+* cd mystery
+* ls
+* cat crimescene
+* //too much crap
+* //grep 'CLUE' crimescene
+* //AAA, DELTA SkyMiles library museum of bash history,
+* //male, 6feet tall
+//Mike Bostock	M	45	Senders Court, line 287
```

Comment:
How did you get Mike Bostock here? There might be more than one suspect that you're missing. Check out other people's solutions to see what they did from here.